libs/remix-lib/src/execution/txExecution.ts

```diff
@@ -83,7 +83,7 @@ export function checkError (execResult, compiledContracts) {
   const error = `Error occured: ${execResult.errorMessage}.\n`
   let msg = ''
   if (exceptionError === errorCode.INVALID_OPCODE) {
```

Comment:
in my case, `exceptionError` was `invalid opcode: PUSH0` so it didn't get in that "if". Is this possible to handle that case? just using `includes` might just do the trick.

Reply:
Done